cmake/README.md

```diff
@@ -16,6 +16,7 @@ The following variables are provided for oneDPL configuration:
 | ONEDPL_USE_AOT_COMPILATION   | BOOL   | Enable ahead-of-time compilation for the GPU or CPU device types                              | OFF           |
 | ONEDPL_ENABLE_SIMD           | BOOL   | Enable SIMD vectorization by passing an OpenMP SIMD flag to the compiler if supported; the flag is passed to user project compilation string if enabled | ON           |
 | ONEDPL_AOT_ARCH              | STRING | Architecture options for ahead-of-time compilation, supported values can be found [here](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top/compilation/ahead-of-time-compilation.html)                                                                                            | "*" for GPU device and "avx" for CPU device |
+| ONEDPL_EXPLICIT_KERNEL_NAMES | STRING | Control usage of Kernel names in tests [AUTO, ALWAYS]: if AUTO then Kernel names are not used in tests when unnamed lambda option is switched on; if ALWAYS then Kernel names are used in tests explicitly | AUTO          |
```

Comment:
As Dmitry said in his comment above, it makes sense to add "TEST" to the names of CMake variables that are supposed to only affect testing. Let's do that at least for this new variable.

@dmitriy-sobolev, for the existing variables, could renaming result in any issues for users (in other words, should it be considered as a potentially breaking change)?

Reply:
Fixed: ```ONEDPL_EXPLICIT_KERNEL_NAMES``` renamed to ```TEST_EXPLICIT_KERNEL_NAMES```
- in C++ code
- in CMake

Reply:
@akukanov, I do not consider it is a breaking change, because those cmake options (`ONEDPL_DEVICE_TYPE` and `ONEDPL_DEVICE_BACKEND`) affect only oneDPL tests.

Reply:
@dmitriy-sobolev so let's create another patch to change the name prefix for the three CMake variables to `ONEDPL_TEST_` to clearly indicate both the project and the purpose.
This proposal does not extend to the corresponding macros in the code - these should be similar but exact match is not the goal. In particular, the `ONEDPL_` prefix should certainly **not** be used for test-only macros,